app/jobs/hyku_addons/reindex_model_job.rb

```diff
@@ -1,43 +1,38 @@
 # frozen_string_literal: true
+
+# call with
+#  use_work_ids an array is just for testing a few ids
+# HykuAddons::ReindexModelJob.perform_now("GenericWork", "repo.hyku.docker", limit: 1, options: { cname_doi_mint: ["repo.hyku.docker"], use_work_ids: ["15b3c16b-8183-4d2a-86ef-d144176fa0c8"]} )
+#
+# HykuAddons::ReindexAvailableWorksJob.perform_later(["repo.hyku.docker"], options: { cname_doi_mint: ["repo.hyku.docker"]} )
+
 module HykuAddons
   class ReindexModelJob < ApplicationJob
-    queue_as :reindex
-
     rescue_from Hyrax::DOI::DataCiteClient::Error, Ldp::Gone, Ldp::HttpError, RSolr::Error::Http, RSolr::Error::ConnectionRefused do |exception|
       Rails.logger.debug exception.inspect
       retry_job(wait: 5.minutes)
     end
 
-    def perform(klass, cname, limit: 25, page: 1, cname_doi_mint: [])
+    def perform(klass, cname, limit: 25, page: 1, options: {})
```

Comment:
There are no tests updated for this class.

Reply:
So what is your suggestion @thenapking

Reply:
This class calls save on some works, presumably in order to trigger reindexing them.  We could at the very least check that the correct works have been updated.  We can also check that the work's attribute `doi_status_when_public` gets set to `findable` if the work has a doi. 
As there are now two options that can be passed in to this job, I would expect there to be at least two test scenarios covering the use of each of the options.

Reply:
Thanks for adding the two test classes! I think this vastly improves the change.